src/components/ThemeModal/index.tsx

```diff
@@ -0,0 +1,89 @@
+import React, { useEffect, useState } from 'react';
+import styled, { useTheme } from 'styled-components';
+
+// Components
+import { BackButton, HeaderTitle, ModalHeader } from '../Menu/MenuModalWrapper';
+
+// Hooks
+import { useEtherspot } from '../../hooks';
+
+// Icons
+import { IoMdCheckmark } from 'react-icons/io';
+import { CloseButton } from '../Button';
+
+// Utils
+import { THEME_TYPES, Theme, getTheme } from '../../utils/theme';
+
+const ThemeModal = ({ onBackButtonClick }: { onBackButtonClick: () => void }) => {
+  const [appliedTheme, setAppliedTheme] = useState<string>('');
+  const theme: Theme = useTheme();
+  const { changeTheme } = useEtherspot();
+  const themeOptions = Object.keys(THEME_TYPES);
```

Comment:
We don't really need this on a separate line. Also we'll need to update this to use the enum